plugins/scanners/fossid/src/main/kotlin/FossIdScanResults.kt

```diff
@@ -148,7 +148,7 @@ private fun mapLicense(
         //       which cannot be parsed to an SpdxExpression. A better solution could be to automatically
         //       convert the strings into a form that can be parsed, then the mapping could be applied globally.
         LicenseFinding(license.mapLicense(detectedLicenseMapping), location)
-    }.onSuccess { licenseFinding ->
+    }.map { licenseFinding ->
```

Comment:
@nnobelis please double-check.

Reply:
We discussed with @mnonnenmacher : it looks wrong and should have been picked up by the `FossIdLicenseMappingTest`. I will look at it in the future.

Reply:
I found why the test was previously passing. Please see https://github.com/oss-review-toolkit/ort/pull/8495.